src/js/view/view.js

```diff
@@ -877,9 +877,12 @@ define([
         }
 
         function _showDisplay() {
+            var hasControls = _model.get('controls');
+            var state = _model.get('state');
+
             var model = _instreamMode ? _instreamModel : _model;
```

Comment:
Doesn't look like you're using model anymore after the refactor.  Can probably get rid of it.